examples/learning/helloworld64.html

```diff
@@ -40,36 +40,35 @@ <h1>'Hello, world!' example</h1>
   // Opening PDF by passing its binary data as a string. It is still preferable
   // to use Uint8Array, but string or array-like structure will work too.
   var loadingTask = pdfjsLib.getDocument({ data: pdfData, });
-  loadingTask.promise.then(function(pdf) {
+  (async function () {
+    var pdf = await loadingTask;
     // Fetch the first page.
-    pdf.getPage(1).then(function(page) {
-      var scale = 1.5;
-      var viewport = page.getViewport({ scale: scale, });
-      // Support HiDPI-screens.
-      var outputScale = window.devicePixelRatio || 1;
+    var page = await pdf.getPage(1);
+    var scale = 1.5;
+    var viewport = page.getViewport({ scale: scale });
+    // Support HiDPI-screens.
+    var outputScale = window.devicePixelRatio || 1;
 
-      // Prepare canvas using PDF page dimensions.
-      var canvas = document.getElementById('the-canvas');
-      var context = canvas.getContext('2d');
+    // Prepare canvas using PDF page dimensions.
+    var canvas = document.getElementById("the-canvas");
+    var context = canvas.getContext("2d");
 
-      canvas.width = Math.floor(viewport.width * outputScale);
-      canvas.height = Math.floor(viewport.height * outputScale);
-      canvas.style.width = Math.floor(viewport.width) + "px";
-      canvas.style.height =  Math.floor(viewport.height) + "px";
+    canvas.width = Math.floor(viewport.width * outputScale);
+    canvas.height = Math.floor(viewport.height * outputScale);
+    canvas.style.width = Math.floor(viewport.width) + "px";
+    canvas.style.height = Math.floor(viewport.height) + "px";
 
-      var transform = outputScale !== 1
-        ? [outputScale, 0, 0, outputScale, 0, 0]
-        : null;
+    var transform =
+        outputScale !== 1 ? [outputScale, 0, 0, outputScale, 0, 0] : null;
 
-      // Render PDF page into canvas context.
-      var renderContext = {
-        canvasContext: context,
-        transform: transform,
-        viewport: viewport,
-      };
-      page.render(renderContext);
-    });
-  });
+    // Render PDF page into canvas context.
+    var renderContext = {
+      canvasContext: context,
+      transform: transform,
+      viewport: viewport,
+    };
+    page.render(renderContext);
+      })();
```

Comment:
unindent

Reply:
@SamyCookie thank you for the review. I would make the required change.